transit-vpc-primary-account-existing-vpc.template

```diff
@@ -80,6 +80,16 @@
       "Type" : "String",
       "Default" : ""
     },
+    "CodeLocationS3Bucket" : {
+      "Description" : "The location of the CISCO code used by this solution",
```

Comment:
is this cisco code or amzn? maybe a term that isn't the vendor but what the code is would fit better?